### ISSUE TITLE
Only variables should be passed by reference, 114

### ISSUE THREAD

**Issue by AlecWeekes** [Lansoweb/LosReCaptcha#8]
The following error has been plaguing my Sentry for the last few months, and I've managed to locate it to the following section;

src > Captcha > ReCaptcha.php:114

```
        if (! $res->isSuccess()) {
            $this->error(self::BAD_CAPTCHA, end($res->getErrorCodes()));
            //$service->setParam('error', $res->getErrorCodes());
            return false;
        }
```

Should we not be passing that to a variable first, and then using that, such as;
```
        if (! $res->isSuccess()) {
            $result = $res->getErrorCodes();
            $this->error(self::BAD_CAPTCHA, end($result));
            //$service->setParam('error', $res->getErrorCodes());
            return false;
        }
```

**Comment by Lansoweb** [comment]
True! Thanks for the report. Will release a fix later today.

**Comment by Lansoweb** [comment]
@AlecWeekes Fixed on [2.1.1](
https://github.com/Lansoweb/LosReCaptcha/releases/tag/2.1.1). Thanks!

**Comment by AlecWeekes** [comment]
@Lansoweb Thanks for dealing with this so swiftly.
